packages/flutter/test/material/slider_theme_test.dart

```diff
@@ -515,25 +489,22 @@ void main() {
 
     expect(
       sliderBox,
-      paints..circle(x: 208, y: 300, radius: 9, color: sliderTheme.thumbColor),
+      paints..circle(x: 212, y: 300, radius: 9, color: sliderTheme.thumbColor),
     );
 
     await tester.pumpWidget(_buildApp(sliderTheme, value: 0.25, enabled: false));
     await tester.pumpAndSettle(); // wait for disable animation
-    // Radius should be 6, or 2/3 of 9. 2/3 because the default disabled thumb
-    // radius is 4 and the default enabled thumb radius is 6.
-    // TODO(clocksmith): This ratio will change once thumb sizes are updated to spec.
     expect(
       sliderBox,
-      paints..circle(x: 208, y: 300, radius: 6, color: sliderTheme.disabledThumbColor),
+      paints..circle(x: 212, y: 300, radius: 9, color: sliderTheme.disabledThumbColor),
     );
   });
 
 
   testWidgets('The default slider tick mark shape size can be overridden', (WidgetTester tester) async {
     final SliderThemeData sliderTheme = ThemeData().sliderTheme.copyWith(
       tickMarkShape: const RoundSliderTickMarkShape(
-        tickMarkRadius: 5
+        tickMarkRadius: 5,
```

Comment:
No need for `,`. Can all fit on one line.

Reply:
Done, nice catch.